ldm/modules/attention.py

```diff
@@ -188,9 +190,11 @@ def forward(self, x, context=None, mask=None):
             sim.masked_fill_(~mask, max_neg_value)
             del mask
 
-        # attention, what we cannot get enough of, by halves
-        sim[4:] = sim[4:].softmax(dim=-1)
-        sim[:4] = sim[:4].softmax(dim=-1)
+        if device_type == 'mps':      #special case for M1 - disable neonsecret optimization
+            sim - sim.softmax(dim=-1)
```

Comment:
Small typo here appears to be the source of the regression. Should be `sim = sim.softmax(dim=-1)`. After making this change it works on my setup. M1 Mac Mini, macOS Monterey 12.5.1, Conda env `environment-mac.yml`, Python 3.10.5.

Reply:
@heurihermilab, could you confirm this fix works for you?

Reply:
Yes, fixing the - for = typo fixes it.  Thanks!

Reply:
yes performace is back  with a little improvment , too 👍 

50/50 [00:48<00:00,  1.04it/s] neonsecret-optimizations
50/50 [00:53<00:00,  1.06s/it] Release 1.13

Reply:
Yes, this works.

Reply:
Great, thanks for finding that! I am going to merge into development now.